### PR TITLE
add oasis

### PR DESCRIPTION
**Pull request by namgold**

### CODE REVIEW

File: src/apollo/client.ts [hunk 1]
```diff
@@ -248,7 +248,7 @@ export const routerUri: { [chainId in ChainId]?: string } = {
   [ChainId.BTTC]: `${process.env.REACT_APP_AGGREGATOR_API}/bttc/route`,
   [ChainId.AURORA]: `${process.env.REACT_APP_AGGREGATOR_API}/aurora/route`,
   [ChainId.VELAS]: `${process.env.REACT_APP_AGGREGATOR_API}/velas/route`,
-  [ChainId.OASIS]: `https://aggregator-api.dev.kyberengineering.io/oasis`, //todo namgold: oasis aggregator
+  [ChainId.OASIS]: `https://aggregator-api.dev.kyberengineering.io/oasis/route`, //todo namgold: oasis aggregator
```
Comment (ngtuna) [inline src/apollo/client.ts:251]: why hardcode url ?

Comment (ngtuna) [reply src/apollo/client.ts:251]: okay got it

File: src/constants/index.ts [hunk 2]
```diff
@@ -488,54 +506,37 @@ export const AVERAGE_BLOCK_TIME_IN_SECS: { [chainId in ChainId]: number } = {
...
-  [ChainId.FANTOM]: new Token(ChainId.MUMBAI, UNI_ADDRESS, 18, 'UNI', 'Uniswap'),
-  [ChainId.CRONOSTESTNET]: new Token(ChainId.CRONOSTESTNET, UNI_ADDRESS, 18, 'UNI', 'Uniswap'),
-  [ChainId.CRONOS]: new Token(ChainId.CRONOS, UNI_ADDRESS, 18, 'UNI', 'Uniswap'),
-  [ChainId.AURORA]: new Token(ChainId.AURORA, UNI_ADDRESS, 18, 'UNI', 'Uniswap'),
-  [ChainId.ARBITRUM_TESTNET]: new Token(ChainId.ARBITRUM_TESTNET, UNI_ADDRESS, 18, 'UNI', 'Uniswap'),
-  [ChainId.ARBITRUM]: new Token(ChainId.ARBITRUM, '0xFa7F8980b0f1E64A2062791cc3b0871572f1F7f0', 18, 'UNI', 'Uniswap'),
-  [ChainId.BTTC]: new Token(ChainId.BTTC, UNI_ADDRESS, 18, 'UNI', 'Uniswap'),
-  [ChainId.VELAS]: new Token(ChainId.VELAS, UNI_ADDRESS, 18, 'UNI', 'Uniswap'),
-}
+// const UNI_ADDRESS = '0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984'
```
Comment (viet-nv) [inline src/constants/index.ts:517]: remove these line

Comment (namgold) [reply src/constants/index.ts]: Done

**Review by viet-nv — APPROVED** [review]

**Pull request merged**